TriblerGUI/tribler_window.py

```diff
@@ -683,11 +685,19 @@ def exit_full_screen(self):
 
     def close_tribler(self):
         if not self.core_manager.shutting_down:
+            def show_force_shutdown():
+                self.loading_text_label.setText("Tribler is taking longer than expected to shut down.")
```

Comment:
Maybe add here: 'You can force Tribler to shutdown by pressing the button below. This might lead to data loss.` ?